radixdlt/src/main/java/com/radixdlt/consensus/liveness/Pacemaker.java

```diff
@@ -26,8 +26,31 @@
  * Interface for an event coordinator to manage the pacemaker
  */
 public interface Pacemaker {
+
+	/**
+	 * Retrieves the local current view the pacemaker is at
+	 * @return view of the pacemaker
+	 */
 	View getCurrentView();
```

Comment:
Seems like this could be an observable too (like last committed vertex) for consistency, but not sure if necessary

Reply:
Don't think so, this has to be synchronous with the rest of the operations. Would be much more difficult if implemented as async